content/blog/monthly-july-19.md

```diff
@@ -0,0 +1,128 @@
+
+You'll find the newsletter [on the TiKV blog](https://tikv.org/blog/) from the first week of each month.
+
+[Let us know](https://github.com/tikv/website/issues/new) what you think!
+
+## News
+
+We [released version 3.0](https://tikv.org/blog/tikv-3.0ga/) of TiKV! This was a huge and exciting release, and is accompanied by the 3.0 release of [TiDB](https://pingcap.com/blog/tidb-3.0-announcement/).
+
+Jepsen tested TiDB and TiKV in a detailed [analysis](https://pingcap.com/blog/tidb-passes-jepsen-test-for-snapshot-isolation-and-single-key-linearizability/). This focused on transaction guarantees and failure testing.
+
+In other news, PingCAP also released 1.0 of TiDB Operator, a tool for deploying TiDB in the cloud. Release announcement in [English](https://pingcap.com/blog/database-cluster-deployment-and-management-made-easy-with-kubernetes/) and [Chinese](https://pingcap.com/blog-cn/tidb-operator-1.0-ga/)
+
+
+## Reading material
+
+Some blog posts and articles related to TiKV:
+
+* The TiKV team launched a [training course](https://tikv.org/blog/talent-training/) covering Rust and distributed systems (English).
+* Nick Cameron wrote about migrating the Rust TiKV client from [futures 0.1 to 0.3](https://www.ncameron.org/blog/migrating-a-crate-from-futures-0-1-to-0-3/) (English).
+* Ana Hobden wrote on the PingCAP blog about [benchmarking distributed databases](https://www.pingcap.com/blog/why-benchmarking-distributed-databases-is-so-hard/) (English).
+* Source code reading posts on [an analysis of control flow through service layers](https://pingcap.com/blog-cn/tikv-source-code-reading-9/), [Snapshot send and receive](https://pingcap.com/blog-cn/tikv-source-code-reading-10/), and [Storage and transaction control](https://pingcap.com/blog-cn/tikv-soucre-code-reading-11/) (Chinese).
+
+
+## Current projects
+
+There's lots going on in the TiKV ecosystem, here we'll try to highlight some of the interesting projects.
+
+### Performance
+
+There's ongoing work to measure and improve TiKV's performance in all areas. We hope to report more on specifics soon. We've been making heavy use of tools like Intel Vtune and flamegraphs to find the slowest parts of TiKV. We hope to report more on specifics soon.
+
+
+### Titan
+
+[Titan](https://github.com/pingcap/titan) is a storage engine which improves on RocksDB. It is part of TiKV already and you can try it out by following [these instructions](https://tikv.org/docs/3.0/tasks/configure/titan/). Note that you can't go back to RocksDB after enabling Titan. We're working to improve its performance and stability. You can read more in this [blog post](https://pingcap.com/blog/titan-storage-engine-design-and-implementation/).
```

Comment:
```suggestion
[Titan](https://github.com/pingcap/titan) is a storage engine plugin which improves on RocksDB. It is part of TiKV already and you can try it out by following [these instructions](https://tikv.org/docs/3.0/tasks/configure/titan/). We're working to improve its performance and stability. You can read more in this [blog post](https://pingcap.com/blog/titan-storage-engine-design-and-implementation/).
```

Reply:
I think we have advertised Titan as a plug-in instead of a storage engine.

Reply:
>  Note that you can't go back to RocksDB after enabling Titan.

This is not true. 
https://tikv.org/docs/3.0/tasks/configure/titan/#how-to-fall-back-to-rocksdb